src/index.ts

```diff
@@ -315,3 +422,103 @@ export function generateSalt(byteCount = 32): string {
   );
   return b64encoded;
 }
+
+/**
+ * Checks if the provided key is an `EncryptionKey`.
+ *
+ * @param encryptionKey - The object to check.
+ * @returns Whether or not the key is an `EncryptionKey`.
+ */
+export function isEncryptionKey(
```

Comment:
Hmm, do these need to be exported? Better to keep them internal if we can, to reduce API surface area and make future refactors easier

Reply:
Good point! I just made these private https://github.com/MetaMask/browser-passworder/pull/49/commits/60ec49af96b2093cc57fd0f046c73d5e75a6fb11